sdk/src/main/java/io/customer/sdk/api/retrofit/CustomerIoCallAdapter.kt

```diff
@@ -1,5 +1,6 @@
 package io.customer.sdk.api.retrofit
```

Comment:
I can't seem to wrap my head around this whole file + it's unit test. I am not understanding what it's used for and what it's doing. Perhaps more descriptive names of classes, functions, variables + docs would help?

Reply:
We have a `CustomerIOCallAdapter` that extends, Retrofit, `CallAdapter` class. 
These tests are around that adapter, to ensure, the conversion, responses, return types are correct.